test/onnx/test_fx_to_onnx_with_custom_op_overload.py

```diff
@@ -0,0 +1,56 @@
+# Owner(s): ["module: onnx"]
+import onnxscript
+import pytorch_test_common
+import torch
+import torch.onnx._internal.fx.custom_operator
+from torch.onnx._internal import fx
+from torch.testing._internal import common_utils
+
+
+class TestFxToOnnx(pytorch_test_common.ExportTestCase):
+    def setUp(self):
+        super().setUp()
+        self.opset_version = torch.onnx._constants.ONNX_DEFAULT_OPSET
+
+    def test_simple_function(self):
+        # Steps to put custom op into onnx model
+        #  1. Define a custom function and register it as torch._ops.OpOverload so that
+        #     this function can pass through PyTorch 2.0's FX-generation pipeline
+        #     (e.g., dynamo.export). Otherwise, the custom function will be traced
+        #     into (or decomposed) and exporter won't see any custom functions.
+        #  2. Write a custom exporter using ONNX Script for this custom function.
+        #  3. Register the custom exporter to the custom function (type: torch._ops.OpOverload).
```

Comment:
I think we should avoid term `exporter`, since it collides with the general onnx exporter.